integr8ly/installation
# INTLY-1268: use supported RHOAR runtimes in launcher

## Additional Information
https://issues.jboss.org/browse/INTLY-1268

Backend image tag has been updated based on advice from JIRA comment:
https://issues.jboss.org/browse/INTLY-1268?focusedCommentId=13709165&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13709165
Together with the filter that is added, it did work as expected.

## Verification Steps
Install Integreatly via this PR
Open Launcher, start new project and click through to Runtimes screen
Check each available runtime and make sure that only Red Hat supported versions are present(no community versions).
